Doc/library/ssl.rst

```diff
@@ -1641,6 +1641,22 @@ to speed up repeated connections from the same clients.
 
    .. versionadded:: 3.6
 
+.. method:: SSLContext.get_groups()
+
+   Get a list of groups implemented for key agreement, taking into account
+   the SSLContext's current TLS ``minimum_version`` and ``maximum_version``
+   values.
+
+   Example::
```

Comment:
```suggestion
   values. For example::
```

And remove the indentation in the example (I don't think it's necessary)

Reply:
Done.